builder_automate.md

```diff
@@ -0,0 +1,389 @@
+
+Copy the any custom certificate `.crt` and `.key` files to the same location as the `./install.sh` script.
+
+1. Locate the SSL certificate and key pair.
+1. Make the keys accessible to Builder, if needed.
+1. If you're testing this workflow, make your own key pair and copy them to `/on-prem-builder`. This example uses a Vagrant VM:
+
+    ```bash
+    sudo openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout /etc/ssl/private/ssl-certificate.key -out /etc/ssl/certs/ssl-certificate.crt %>
+
+    sudo cp /etc/ssl/private/ssl-certificate.key .
+    sudo cp /etc/ssl/certs/ssl-certificate.crt .
+    sudo chown vagrant:vagrant ssl-certificate.*
+    ````
+
+1. You can confirm that the keys were copied:
+
+    ```bash
+    cat ./ssl-certificate.key
+    cat ./ssl-certificate.crt
+    ```
+
+### Step Four: Install Builder
+
+1. Run the install script. This installs both Chef Habitat Builder on-prem and the Chef Habitat datastore:
+
+    ```bash
+    bash ./install.sh
+    ```
+
+1. Accept both licenses.
+1. All services should report back as `up`. It make take a few minutes to come up.
+
+    ```bash
+    sudo hab svc status
+    ```
+
+    Should return something similar to:
+
+    ```output
+    package                                        type        desired  state  elapsed (s)  pid    group
+    habitat/builder-api/8473/20190830141422        standalone  up       up     595          28302  builder-api.default
+    habitat/builder-api-proxy/8467/20190829194024  standalone  up       up     597          28233  builder-api-proxy.default
+    habitat/builder-memcached/7728/20180929144821  standalone  up       up     597          28244  builder-memcached.default
+    habitat/builder-datastore/7809/20181019215440  standalone  up       up     597          28262  builder-datastore.default
+    habitat/builder-minio/7764/20181006010221      standalone  up       up     597          28277  builder-minio.default
+    ```
+
+### Step Five: Copy Automate's Certificate to the `/hab/cache/ssl` Directory
+
+1. View and copy the Chef Automate certificate. Change the server name to your Chef Automate installation:
+
+    ```bash
+    openssl s_client -showcerts -servername chef-automate.test -connect chef-automate.test:443 < /dev/null | openssl x509
+    ```
+
+    Copy the output to an accessible file.
+
+    ```output
+    # Copy the contents including the begin and end certificate
+    # -----BEGIN CERTIFICATE-----
+    # Certificate content here
+    #-----END CERTIFICATE-----
+    ```
+
+1. Make a file for you cert at `/hab/cache/ssl/`, such as `automate-cert.crt`. For a `.pem` file, `automate-cert.pem`. Overwriting `cert.pem` will cause your Builder installation to fail.
+1. Paste the Chef Automate certificate into your file, `/hab/cache/ssl/automate-cert.crt`
+1. Restart builder
+
+   ```bash
+   sudo systemctl restart hab-sup
+   ``` %>
+
+### You're Done!
+
+1. Login at `https://chef-builder.test`
+
+## Troubleshooting
+
```

Comment:
Only the "### Unable to retrieve OAuth token" section seems relevant to Automate.
The other t-shooting steps should just refer to the README (and update the README if any general changes are needed)